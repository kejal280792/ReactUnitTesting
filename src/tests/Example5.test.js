import React from 'react';
import renderer from 'react-test-renderer';
import Example5 from '../Examples/Example5';

describe('DisplayName', () => {
  test('Should render Vrushali', () => {
    const component = renderer.create(<Example5 name={'Vrushali Test 123'} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
