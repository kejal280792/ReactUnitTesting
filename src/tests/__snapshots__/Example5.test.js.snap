// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayName Should render Vrushali 1`] = `
<div>
  Welcome 
  Vrushali Test 123
   to React World!
</div>
`;
